modules/docs/arrow-docs/docs/docs/effects/fx/polymorphism/README.md

```diff
@@ -194,8 +195,8 @@ import arrow.data.extensions.list.fx.fx
 //sampleStart
 val result1 = unsafe { 
   fx {
-    val (a) = listOf(1, 2)
-    val (b) = listOf(true, false)
+    val (a) = listOf(1, 2)          // This is destructuring the list and is obtaining the first element,
+    val (b) = listOf(true, false)   // that's why the output is `ListK(list=[Tuple2(a=1, b=true)])`
```

Comment:
@pakoito @raulraja I don't totally understand this example, is this behaviour on purpose?

Reply:
yeah, this is wrong, because List has it's own component1. we should use ! or bind here to fix the example and this will be ready to go. Thanks!

Reply:
@raulraja Found the issue, the binding wasn't properly executed because the `.k()` call on the lists was missing.